src/main/java/de/doubleslash/keeptime/REST_API/controller/ProjectController.java

```diff
@@ -0,0 +1,213 @@
+// Copyright 2024 doubleSlash Net Business GmbH
+//
+// This file is part of KeepTime.
+// KeepTime is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+package de.doubleslash.keeptime.REST_API.controller;
+
+import de.doubleslash.keeptime.REST_API.DTO.ProjectColorDTO;
+import de.doubleslash.keeptime.REST_API.DTO.WorkDTO;
+import de.doubleslash.keeptime.REST_API.mapper.ProjectMapper;
+import de.doubleslash.keeptime.REST_API.mapper.WorkMapper;
+import de.doubleslash.keeptime.controller.Controller;
+import de.doubleslash.keeptime.model.Model;
+import de.doubleslash.keeptime.model.Project;
+import de.doubleslash.keeptime.model.Work;
+import de.doubleslash.keeptime.model.repos.ProjectRepository;
+import de.doubleslash.keeptime.model.repos.WorkRepository;
+import org.springframework.dao.DataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.validation.Valid;
```

Comment:
does not compile on my machine
![image](https://github.com/doubleSlashde/KeepTime/assets/5772511/b3a8e232-039e-4fef-aca6-8c306bec5a8e)

Reply:
u now added java.validation dependency. But in other places, where you added validation u do use jakarta dependency. e.g. https://github.com/doubleSlashde/KeepTime/pull/173/files#diff-b69f18a1499af103e893d99780a4038bcfc627c1fc2c40f90d56d9ffacaf0d77R22
so please use the jakarta validation everywhere